nebula_decoders/src/nebula_decoders_continental/decoders/continental_ars548_decoder.cpp

```diff
@@ -0,0 +1,574 @@
+  radar_status_.timestamp_seconds = sensor_status_packet.stamp.timestamp_seconds.value();
+
+  if (sensor_status_packet.stamp.timestamp_sync_status == 1) {
+    radar_status_.timestamp_sync_status = "SYNC_OK";
+  } else if (sensor_status_packet.stamp.timestamp_sync_status == 2) {
+    radar_status_.timestamp_sync_status = "NEVER_SYNC";
+  } else if (sensor_status_packet.stamp.timestamp_sync_status == 3) {
+    radar_status_.timestamp_sync_status = "SYNC_LOST";
+  } else {
+    radar_status_.timestamp_sync_status = "INVALID_VALUE";
+  }
+
+  radar_status_.sw_version_major = sensor_status_packet.sw_version_major;
+  radar_status_.sw_version_minor = sensor_status_packet.sw_version_minor;
+  radar_status_.sw_version_patch = sensor_status_packet.sw_version_patch;
+
+  radar_status_.plug_orientation = sensor_status_packet.status.plug_orientation == 0 ? "PLUG_RIGHT"
+                                   : sensor_status_packet.status.plug_orientation == 1
+                                     ? "PLUG_LEFT"
+                                     : "INVALID_VALUE";
+
+  radar_status_.max_distance = sensor_status_packet.status.maximum_distance.value();
+
+  radar_status_.longitudinal = sensor_status_packet.status.longitudinal.value();
+  radar_status_.lateral = sensor_status_packet.status.lateral.value();
+  radar_status_.vertical = sensor_status_packet.status.vertical.value();
+  radar_status_.yaw = sensor_status_packet.status.yaw.value();
+
+  radar_status_.pitch = sensor_status_packet.status.pitch.value();
+  ;
+  radar_status_.length = sensor_status_packet.status.length.value();
+  ;
+  radar_status_.width = sensor_status_packet.status.width.value();
+  radar_status_.height = sensor_status_packet.status.height.value();
+  radar_status_.wheel_base = sensor_status_packet.status.wheelbase.value();
+
+  if (sensor_status_packet.status.frequency_slot == 0) {
+    radar_status_.frequency_slot = "0:Low (76.23 GHz)";
+  } else if (sensor_status_packet.status.frequency_slot == 1) {
+    radar_status_.frequency_slot = "1:Mid (76.48 GHz)";
+  } else if (sensor_status_packet.status.frequency_slot == 2) {
+    radar_status_.frequency_slot = "2:High (76.73 GHz)";
+  } else {
+    radar_status_.frequency_slot = "INVALID VALUE";
+  }
+
+  radar_status_.cycle_time = sensor_status_packet.status.cycle_time;
+  radar_status_.time_slot = sensor_status_packet.status.time_slot;
+
+  radar_status_.hcc = sensor_status_packet.status.hcc == 1 ? "Worldwide"
+                      : sensor_status_packet.status.hcc == 2
+                        ? "Japan"
+                        : ("INVALID VALUE=" + std::to_string(sensor_status_packet.status.hcc));
+
+  radar_status_.power_save_standstill =
+    sensor_status_packet.status.powersave_standstill == 0   ? "Off"
+    : sensor_status_packet.status.powersave_standstill == 1 ? "On"
+                                                            : "INVALID VALUE";
```

Comment:
This would be cleaner as a switch statement

Reply:
Addressed in [800b351](https://github.com/tier4/nebula/pull/123/commits/800b351ad5c941b14e31507c4cd59eb6b42ddd4a)